newsletter-april-2021.rst

```diff
@@ -43,24 +51,37 @@ New features
    and `started recommending MyST for writing Markdown with
    Sphinx <https://docs.readthedocs.io/en/stable/intro/getting-started-with-sphinx.html#using-markdown-with-sphinx>`__.
 
```

Comment:
We should link to our changelog here: https://docs.readthedocs.io/page/changelog.html
```suggestion

You can always see the latest changes to our platforms in our `Read the Docs Changelog <https://docs.readthedocs.io/page/changelog.html>`_ and `Ethical Ad Server Changelog <https://ethical-ad-server.readthedocs.io/page/developer/changelog.html>`_.
```